src/components/admin/AddProduct.tsx

```
@@ -0,0 +1,270 @@
+} from "@/components/ui/select"
+import { Textarea } from "@/components/ui/textarea"
+
+// Define the form schema
+const formSchema = z.object({
+  name: z.string().min(2, {
+    message: "Product name must be at least 2 characters.",
+  }),
+  description: z.string().min(10, {
+    message: "Description must be at least 10 characters.",
+  }),
+  price: z.string().refine((val) => !isNaN(Number(val)), {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)